contracts/ColonyStorage.sol

```diff
@@ -204,4 +229,13 @@ contract ColonyStorage is CommonStorage, ColonyDataTypes, ColonyNetworkDataTypes
   function domainExists(uint256 domainId) internal view returns (bool) {
     return domainId > 0 && domainId <= domainCount;
   }
+
+  // Gas optimization so 0 (default) unpacks to WAD -- so underflows are ok here
```

Comment:
Happy to rename, but I do think that doing the arithmetic in one place rather than in multiple places in the contracts is important, since it reduces the chance of error in the future.

Reply:
The renaming is most important, in my eyes, but at what point is the arithmetic so trivial that it doesn't need its own function?

Reply:
It's more that it occurs in several places (2-3 currently) -- if there was one location where these transformations occur, then no problem let's write it directly. The issue is that since this arithmetic occurs in multiple places, having to write it directly increases (significantly, I think) the chance of an inconsistency. I also like that the details of the implementation are obscured, which seems more in line with the abstraction boundaries we might want.

On the other hand, though, one could argue that it's unlikely we will _add_ this packing/unpacking to more places, so maybe implementing it directly in the handful of places we use it isn't so bad, and it'll save two functions.